app/models/post.rb

```diff
@@ -0,0 +1,12 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
+  has_many :comment
+
+  def update_users_posts=(count)
+    user.update_attribute 'posts_counter', count
+  end
```

Comment:
- [ ]  I would recommend to increment counter every time by 1, to do that you can simply use:
```ruby
  def update_post_counter
    user.increment!(:posts_counter)
  end
```
- [ ] OPTIONAL:  use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private:

```ruby
# app/models/post.rb
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```